modules/virt-creating-storage-cluster-fusion-access-san.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-storage-cluster-fusion-access-san_{context}"]
+= Creating a storage cluster with {IBMFusionSAN}
+
+Once you have installed the {IBMFusionSAN} Operator, you can create a storage cluster with shared storage nodes.
+
+The wizard for creating the storage cluster in the {product-title} web console provides easy to follow steps, and lists the relevant nodes with shared disks.
+
+.Prerequisites
+
+* You have bare-metal worker nodes with visible and attached shared LUNs.
```

Comment:
Maybe this is not the best place, but could we add a short explanation like: 
A shared LUN (Logical Unit Number) is a shared disk that is accessed by all workers simultaneously.

Reason for adding it, is that we had a couple of folks getting the wrong idea.